config/initializers/inflections.rb

```diff
@@ -1,6 +1,10 @@
 # frozen_string_literal: true
 
 ActiveSupport::Inflector.inflections do |inflect|
+  inflect.acronym 'AAID'
```

Comment:
J/C, What's AAID? Not in [`acronyms`](https://github.com/department-of-veterans-affairs/acronyms)